tsconfig.json

```diff
@@ -16,53 +16,53 @@
   "compilerOptions": {
     /* Basic Options */
     "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "esnext",                    /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2015", "dom"],              /* Specify library files to be included in the compilation. */
-    "allowJs": true,                       /* Allow javascript files to be compiled. */
+    "module": "esnext",                       /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "lib": ["es2015", "dom"],                 /* Specify library files to be included in the compilation. */
+    "allowJs": true,                          /* Allow javascript files to be compiled. */
     // "checkJs": true,                       /* Report errors in .js files. */
-    "jsx": "react",                        /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    "jsx": "react",                           /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
     // NOTE: enable declaration once we can disable "allowJs"
     // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
     // "declarationDir": "dist",
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "dist",                      /* Redirect output structure to the directory. */
+    "outDir": "dist",                         /* Redirect output structure to the directory. */
     // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     "pretty": true,
     // "composite": true,                     /* Enable project compilation */
-    "removeComments": true,                /* Do not emit comments to output. */
-    // "noEmit": true,                        /* Do not emit outputs. */
```

Comment:
I think this broke storybook...or something else I don't remember exactly. There was a problem with this option as far as I could remember.

We should check that everything works, maybe with the babel setup it's not a problem anymore. Just wanted to raise awareness about this.

Reply:
Ah, yeah thänks. `noEmit` now can be used as babel removes the types and actually creates the bundle. While before when set to `true` the compiler would not yield anything to webpack hence breaking storybook.

I fixed the option as suggested in the `tsconfig.declarations.json` so we emit there.